Tests/BaseObjectTests.swift

```diff
@@ -16,181 +18,183 @@ class BaseObjectTests: QuickSpec {
 
-                    expect(value) == "Expected"
-                }
+                        baseObject.on(eventName) { userInfo in
+                            value = userInfo?["new_value"] as! String
+                        }
+                        baseObject.trigger(eventName, userInfo: ["new_value": "Expected"])
 
-                it("Callback should be called for every callback registered") {
-                    baseObject.on(eventName, callback: callback)
-
-                    var secondCallbackWasCalled = false
-                    baseObject.on(eventName) { _ in
-                        secondCallbackWasCalled = true
+                        expect(value) == "Expected"
                     }
 
-                    baseObject.trigger(eventName)
+                    it("executes multiple callback functions") {
+                        var secondCallbackWasCalled = false
 
-                    expect(callbackWasCalled) == true
-                    expect(secondCallbackWasCalled) == true
-                }
+                        baseObject.on(eventName, callback: callback)
+                        baseObject.on(eventName) { _ in
+                            secondCallbackWasCalled = true
+                        }
+                        baseObject.trigger(eventName)
 
-                it("Callback should not be called for another event trigger") {
-                    baseObject.on(eventName, callback: callback)
+                        expect(callbackWasCalled) == true
+                        expect(secondCallbackWasCalled) == true
+                    }
 
-                    baseObject.trigger("another-event")
+                    it("doesn't execute callback for another event") {
+                        baseObject.on(eventName, callback: callback)
+                        baseObject.trigger("another-event")
 
-                    expect(callbackWasCalled) == false
-                }
-
-                it("Callback should not be called for another context object") {
-                    let anotherObject = BaseObject()
+                        expect(callbackWasCalled) == false
+                    }
 
-                    baseObject.on(eventName, callback: callback)
+                    it("doesn't executes callback for another context object") {
+                        let anotherBaseObject = ConcreteBaseObject()
 
-                    anotherObject.trigger(eventName)
+                        baseObject.on(eventName, callback: callback)
+                        anotherBaseObject.trigger(eventName)
 
-                    expect(callbackWasCalled) == false
+                        expect(callbackWasCalled) == false
+                    }
                 }
-            }
 
-            describe("once") {
-                it("Callback should be called on event trigger") {
-                    baseObject.once(eventName, callback: callback)
-                    baseObject.trigger(eventName)
+                describe("once") {
```

Comment:
According to the Project conventions, we should add the # prefix on the Method Name.

``context("#once")``